### PR TITLE
more strict namespacing for row class, closes #127

### LINKED ISSUE #127: Increasing the width of rendered HTML in Shiny
Hi, thanks very much for your help on an earlier issue #125  .  Taking your example for rendering HTML...

`HTML(
        as.character(
          diffPrint(1:3, 2:4, format='html',style=list(html.output='diff.w.style')
) ) )})`

I see that it fills half the width of the output space.  I have tried changing the disp.width parameter but without success.  I am wondering if there are other ways to increase the width so as to display more contents per line of space?  

Thanks very much for your help on this.

### PR DESCRIPTION
**Pull request by cpsievert**

### CODE REVIEW

File: inst/doc/embed.html [hunk 1]
```diff
@@ -680,9 +680,9 @@ <h2>Easy but Ugly</h2>
...
 </div>
-<p>This is an ugly implementation because it produces illegal HTML. The styles are directly embedded in the body of the document, outside of the HEAD tags. Although this is illegal HTML, it seems to work in most browsers. Another problem is that every diff you use in your document will inject the same CSS code over and over.</p>
+<p>This is an ugly implementation because it produces illegal HTML. The styles are directly embedded in the body of the document, outside of the HEAD tags. Although this is illegal HTML, it seems to work in most bdiffobj-rowsers. Another problem is that every diff you use in your document will inject the same CSS code over and over.</p>
```
Comment (brodieG) [inline inst/doc/embed.html]: Looks like a case of over-eager search-and-replace here.

**Comment by brodieG** [comment]
Thanks for taking the time to debug this.  Figuring out CSS conflicts is one of my least favorite activities.

The PR looks good, but I have a couple of requests:

1. Do you mind making the PR against the development branch (this is in CONTRIBUTING.md, though I'll admit I've failed to read the CONTRIBUTING.md in other projects myself in the past)?
2. Fix the over-eager search-replace I commented on?
3. Add an entry to NEWS.md crediting yourself?

Alternatively, I'm happy to just make the changes directly myself, but obviously then the only credit you get is what I put in NEWS instead of showing up as a contributor.  Let me know how you prefer to go forward with this.

File: inst/doc/embed.html (hunk 1 above)
Comment (brodieG) [reply inst/doc/embed.html]: Actually, this doesn't matter since you did not commit the vignette source itself.

**Comment by cpsievert** [comment]
Go ahead and make the changes yourself and credit me however you wish

**Comment by brodieG** [comment]
Ok, will do, thanks for your help.

**Pull request closed without merging**